Updating Customer Feedback section (#306)
mattermost/mattermost-handbook: operations/customer-success/README.md

## Customer Success

### Customer Feedback

## Overview

> Customer Feedback calls are conducted by all customer facing employees at Mattermost. The objective of a Customer Feedback call is to gather the following information from a customer.

Reviewer: ```suggestion
Customer Feedback calls are conducted by all customer-facing employees at Mattermost. The objective of a Customer Feedback call is to gather the following information:
```

- Feature requests
- Integration usage
- Use cases
- Understand which teams use Mattermost
- Product design review feedback

> - Challenges solving

Reviewer: ```suggestion
- Challenges faced
```

These calls are logged in Salesforce for tracking and reporting purposes.  

## Logging calls in Salesforce

1. Log into Salesforce
2. Type in account name in the search bar located in the middle of the screen
3. Select the account

> 4. Hover over activity history

Reviewer: ```suggestion
4. Hover over **Activity History**
```

> 5. Select “log a call” in the upper right-hand side of the screen

Reviewer: ```suggestion
5. Select **Log a Call** in the top-right corner of the screen
```

6. Fill out the following sections

>    Subject - Quick description of the call.  IE: Customer Reply Thread Design Review

Reviewer: ```suggestion
   **Subject:** Short description of the call (e.g. "Customer Reply Thread Design Review")
```

>    Name: Customer contact(s) that attended the call

Reviewer: ```suggestion
   **Name:** Customer contact(s) who attended the call
```

>    Type: Was this a call or meeting (in-person)

Reviewer: ```suggestion
   **Type:** Whether this was a call or in-person meeting
```

>    Related To: Account name

Reviewer: ```suggestion
   **Related To:** Account name
```

>    Sub-Type: Choose "Customer Feedback"

Reviewer: ```suggestion
   **Sub-Type:** Choose **Customer Feedback**
```

>    Due Date: Auto-populated to today's date

Reviewer: ```suggestion
   **Due Date:** Auto-populated to today's date
```

>    Comments: Details of the call or meeting

Reviewer: ```suggestion
   **Comments:** Details of the call or meeting
```

>  7. Select Save

Reviewer: ```suggestion
 7. Select **Save**
```
